### PR TITLE
pkp/pkp-lib#7846 Update removeLocaleKey to use provided directory path

### PR DESCRIPTION
**Pull request by ewhanson**

### CODE REVIEW

File: tools/removeLocaleKey.php [hunk 1]
```diff
@@ -39,8 +39,8 @@ public function __construct($argv = [])
 
         $this->localeKey = array_shift($argv);
 
-        if (sizeof($this->argv) > 2) {
-            $this->dirs = $argv;
+        if (sizeof($argv) == 1) {
+            $this->dirs = [array_shift($argv)];
```
Comment (asmecher) [inline tools/removeLocaleKey.php:43]: I think this doesn't work if more than one path is specified. I think the only fix required is changing the `if` condition from
```php
if (sizeof($this->argv) > 2) {
    $this->dirs = $argv;
}
```
...to...
```php
if (!empty($this->argv)) {
    $this->dirs = $argv;
}
```

Comment (ewhanson) [reply tools/removeLocaleKey.php:43]: I see, that makes sense. Looking at it again, it looks like setting it to
```PHP
if (sizeof($this->argv) > 1) {
    $this->dirs = $argv;
}
```
would take care of things as the file path is removed in `$this-argv`, right?

Comment (asmecher) [reply tools/removeLocaleKey.php:43]: Sure, that's equivalent.

Comment (asmecher) [reply tools/removeLocaleKey.php:43]: Sorry, I misspoke -- `> 1` is still wrong. The `array_shift` already popped the key off `$argv` so all that should be left is the directory names if any were specified.

Comment (ewhanson) [reply tools/removeLocaleKey.php]: Right! Sorry to make this more confusing than it needs to be. Was getting a bit hung up on the class property `$argv` vs what was passed into the constructor. I've updated the PR.

Comment (asmecher) [reply tools/removeLocaleKey.php]: Not a problem! Merged, thanks.

**Pull request merged**